git/00.setup.md

```diff
@@ -19,30 +19,6 @@ brew install hub
 alias git="hub"
 ```
 
-## Additional config
-
-It is recommended set the follow configurations for git to make your life
-easier.
-
-Set your default push to tracking. Avoid having to type extra characters and
-prevent accidentally pushing feature branch to master it recommended to set your
-default push setting to tracking using the following.
-
-```bash
-git config --global push.default tracking
-```
-
-Next time you want to push a just run `git push`, if you haven't pushed the
```

Comment:
This still applies when using the `simple` strategy, right? 

http://stackoverflow.com/questions/23918062/simple-vs-current-push-default-in-git-for-decentralized-workflow